DigitalRain/Raindrops/GlitchedRaindrop.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+
+namespace DigitalRain.Raindrops
+{
+    using Columns;
+
+    public class GlitchedRaindrop : IRaindrop
```

Comment:
DOPE name

Reply:
```suggestion
    public class GlitchedRaindrop : IRaindrop, IGameObject
```

Reply:
I now see that IRaindrop does indeed already define an implementation of `IGameObject`. Nice